### DISCUSSION TITLE
Highest jQuery version for Bootstrap 4.6.2

### DISCUSSION THREAD

**Discussion by jamespughwales** [twbs/bootstrap#37212] [Q&A]
Hi, we're looking to update the jQuery version on a website to the highest stable release 3.6.1, noticed the BS4 website mentions to use: https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js

(https://getbootstrap.com/docs/4.6/getting-started/introduction/)

**Would anyone be able to help/know what the highest supported jQuery version is for Boostrap 4.6.2?** (current latest)

I'm looking to find out if jquery 4.6.2 will work perfectly or whether we need to look into migrating to Bootstrap 5.

Thanks

**Comment by coliff** [comment]
hey! It's my understanding that the only reason that the Bootstrap v4 docs are referencing and using jQuery 3.5.1 (and not the latest 3.6.x) is because IE10 tests were failing.

See https://github.com/twbs/bootstrap/pull/33243 for details.

As IE10 usage is miniscule and has been deprecated by Microsoft for a long time, you should be completely fine to use jQuery 3.6.1 with Bootstrap 4.6.2 with no issues at all.

**Reply by Edditoria** [reply]
Agree with this. This is a good reason to use Bootstrap v4 for website to support special need.

**Comment by JMLucas96** [comment]
It supports jQuery 3.7.1? Thank you!!

**Reply by coliff** [reply]
Yes, jQuery 3.7.1 is supported!
